Anytype/Sources/PresentationLayer/RelationValueFlow/RelationValueCoordinator.swift

```diff
@@ -44,6 +48,24 @@ final class RelationValueCoordinator: RelationValueCoordinatorProtocol,
             return
         }
         
+        if FeatureFlags.newDateRelationCalendarView, case .date(let date) = relation {
```

Comment:
Maybe create new relation coordinator view and migrate redesigned relations to swiftui navigation?

Reply:
I plan to make it with new feature relations block 👌